publish.js

```diff
@@ -11,9 +12,9 @@ const remark = require('remark');
 const u = require('unist-builder');
 const fs = require('mz/fs');
 const fecha = require('fecha');
-require('any-observable/register/rxjs-all');
-const Observable = require('any-observable');
-const streamToObservable = require('stream-to-observable');
+const {merge, throwError} = require('rxjs');
+const {catchError, filter, finalize} = require('rxjs/operators');
+const streamToObservable = require('@samverschueren/stream-to-observable');
```

Comment:
What is the difference between these implementations of `streamToObservable`?

Reply:
The listr package is used as the publish task runner. `@samverschueren/stream-to-observable` is already a dependency of that package. It seems to be more maintained than the forked package. Not sure if it is needed though.